packages/voyager-metrics/src/metrics.ts

```diff
@@ -36,6 +36,19 @@ const conflictsMetric = new Prometheus.Counter({
   labelNames: ['operation_type', 'name']
 })
 
+const uniqueClientsMetric = new Prometheus.Counter({
+  name: 'unique_clients',
+  help: 'Number of unique clients'
+})
+
+const uniqueUsersMetric = new Prometheus.Counter({
+  name: 'unique_users',
+  help: 'Number of unique users'
+})
+
+const uniqueClientIds = new Set()
```

Comment:
I have a question @aliok. What is the plan to support multiple instances of this service? Is there some kind of aggregation in Prometheus/Grafana that will add these numbers together across instances?

I have a feeling that we will have false readings for `uniqueClientIds` and `uniqueUserIds` if we're adding these numbers together across multiple instances of services. Would it make more sense to log client/user IDs via audit logs or something else and then have that information counted and deduplicated at a top level by the metrics service?

Reply:
Great point. Prometheus service visits every pod and asks this information but still  your point is valid. Let me think about this.

Reply:
I saw this on SO: https://stackoverflow.com/questions/51882134/prometheus-query-to-count-unique-label-values. So I think if we just sent a counter metric data with the label: `clientId=<value>`, and the we can just count it like this:

```
count( count by (clientId) (unique_clients))
```

Reply:
Yes, this works. Changing the code and sending a new PR now.
thx @wei-lee